npm/oxlint/bin/oxlint

```diff
@@ -57,57 +57,40 @@ const isMuslFromChildProcess = () => {
 
 const { platform, arch, env, version, release } = process;
 
-let binPath;
+const PLATFORMS = {
```

Comment:
If you think this change is unnecessary or too unrelated let me know so I can either revert it or extract it into another PR.